aiida/cmdline/commands/cmd_export.py

```diff
@@ -212,5 +227,8 @@ def migrate(input_file, output_file, force, silent, archive_format, version):
             with tarfile.open(output_file, 'w:gz', format=tarfile.PAX_FORMAT, dereference=True) as archive:
                 archive.add(folder.abspath, arcname='')
 
+        if in_place:
+            os.rename(output_file, input_file)
```

Comment:
Will all this work if the file is not in the current working directory and such? I guess it should, since `rename` works with relative paths...

Reply:
I think so. Click checks that the input file is a valid path; and the output file should be an absolute path in this case.